cryostatio/cryostat-web
# feat(dashboard): allow user-defined dashboard card sizes

Signed-off-by: Max Cao <macao@redhat.com>

# Welcome to Cryostat! 👋
## Before contributing, make sure you have:
_______________________________________________

Fixes: #729 

## Description of the change:
Allows cards to be resizable and for these configs to be persisted.

## Motivation for the change:
See https://github.com/cryostatio/cryostat-web/issues/729

## How to manually test:
1. Run normally and add various card configs.
2. Resize some cards by dragging the right side of them.
3. Notice the sizes are persisted when reloading or navigating elsewhere.
